web/components/roomConfig/RoomConfig.tsx

```diff
@@ -0,0 +1,124 @@
+import { h, FunctionComponent, Fragment } from "preact";
+import { useState } from "preact/hooks"
+import { ListItem } from "../ListItem";
+import BridgeAPI from "../../BridgeAPI";
+import ErrorPane from "../ErrorPane";
+import style from "./RoomConfig.module.scss";
+import { GetConnectionsResponseItem } from "../../../src/provisioning/api";
```

Comment:
Importing from `../../../src` looks like an import that should be illegal or you're ending up with the back-end code in your front-end bundle at some point.

Reply:
I *believe* this is fine as long as we're importing interfaces and not classes, at compile time this shouldn't get bundled.

Now we should probably have a separate package for interfaces to avoid this ever being contaminated though.